subsys/net/ip/net_shell.c

```diff
@@ -31,9 +31,7 @@
 #include "ipv6.h"
 #endif
 
-#if defined(CONFIG_HTTP)
```

Comment:
Why remove the CONFIG_HTTP check here?

Reply:
That sneaked in while rebasing the set, I will fix this one.